src/libcore/ops.rs

```diff
@@ -768,7 +768,7 @@ macro_rules! neg_impl_unsigned {
 // neg_impl_unsigned! { usize u8 u16 u32 u64 }
 neg_impl_numeric! { isize i8 i16 i32 i64 i128 f32 f64 }
 
-/// The `Not` trait is used to specify the functionality of unary `!`.
+/// The unary logical negation operator `!`.
```

Comment:
Is the word “unary” really necessary here? Unlike `-` where there are binary and unary uses `!` is always unary. Is the logical negation even defined in some other systems for multiple operands?